README.rst

```diff
@@ -32,7 +32,7 @@ Status shields
     :alt: Benchmarks
 
 * .. image:: https://readthedocs.org/projects/gammapy/badge/?version=latest
-    :target: https://readthedocs.org/projects/gammapy/?badge=latest
+    :target: https://readthedocs.io/projects/gammapy/?badge=latest
```

Comment:
Oh, I didn't notice. I think it must point to the actual docs, not sure why it had the badge there. I'll amend the commit.